domain/src/main/kotlin/domain/actions/SwitchAllOffAction.kt

```diff
@@ -0,0 +1,34 @@
+package domain.actions
+
+import arrow.core.Either
+import arrow.core.flatMap
+import arrow.core.right
+import domain.Status
+import domain.ThingToDevice
+import domain.actions.errors.ActionError
+import domain.asIdOnDevice
+import domain.client.SwitchClient
+import domain.repository.DeviceRepository
+
+class SwitchAllOffAction(
+    private val deviceRepository: DeviceRepository,
+    private val switchClient: SwitchClient
+) {
+
+    fun switchOff(things: List<ThingToDevice>): Either<ActionError, Unit> {
+        things.forEach { thingToDevice ->
+            switchClient.switch(
+                thingToDevice.deviceHost,
+                1.asIdOnDevice(),   //TODO: This is a hardcoded value, can we maintain it?
```

Comment:
MAJOR: I should retrieve the id on device.. for the moment is not needed because every device has only one thing